power_grid_model_io_native_c/power_grid_model_io_native_c/include/power_grid_model_io_native_c/vnf_pgm_converter.h

```diff
@@ -11,10 +11,31 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+
+/**
+ * @brief Create the PGM_VNF_converter
+ * @param handle
+ * @param file_buffer A pointer to the file buffer object which holds the .vnf data.
```

Comment:
Following the logic of the deserializer in the PGM C API, are we expecting here file_buffer to be a byte stream, a string with the data or possibly both? Because two separate calls may be needed (Take a look at `PGM_create_deserializer_from_binary_buffer` and `PGM_create_deserializer_from_null_terminated_string`.

Also, I would re-phrase this line of the docstring to be more aligned with those calls of the C API.

Reply:
We are not expecting both. I can not locate the example file now, however I believe we are expecting a string with the data.

Reply:
Since you are only expecting a C string then, please update the docstring to reflect this is a pointer to the null-terminated C string instead of a (more general) file buffer.